src/Raven.Client/Util/TcpUtils.cs

```diff
@@ -106,7 +107,13 @@ public static async Task<TcpClient> ConnectAsync(string url, TimeSpan? timeout =
             return tcpClient;
         }
 
-        internal static async Task<Stream> WrapStreamWithSslAsync(TcpClient tcpClient, TcpConnectionInfo info, X509Certificate2 storeCertificate,
+        internal static async Task<Stream> WrapStreamWithSslAsync(
+            TcpClient tcpClient,
+            TcpConnectionInfo info,
+            X509Certificate2 storeCertificate,
+#if !(NETSTANDARD2_0 || NETCOREAPP2_1)
+            CipherSuitesPolicy cipherSuitesPolicy,
```

Comment:
Am I missing something? I don't see where this is being used.

Reply:
Thanks